lib/http_server.js

```diff
@@ -20,6 +20,7 @@ var deviceFormatter = require('./api_formats/siren/device.siren');
 var rels = require('zetta-rels');
 
 var querytopic = require('./query_topic');
+console.log(url);
```

Comment:
Be sure to remove this log statement when you get a chance :) Otherwise looking good pending that issue with a test.